### PR TITLE
no need to go get anything. since we use vendoring

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

**Comment by Dieterbe** [comment]
shaves off about 25s from circleci builds

**Comment by Dieterbe** [comment]
how is this? I think `go vet` tends be sometimes a tad too strict.
for example it always complains about all our instantiations of schema.Point:
```
					Datapoints: []schema.Point{
						{123, 60},
						{10000, 120},
						{0, 180},
						{1, 240},
					},
```

it wants us to specify explicitly the keys in those fields, which makes everything soo verbose :-1:

**Comment by woodsaj** [comment]
`go vet` is not too strict.  It enforces best practices.
```
composite literals whose type is defined in a separate package should use the keyed notation.
```

But fixing the code is outside the scope of this change so i have created a separate issue.
https://github.com/raintank/metrictank/issues/388

File: Makefile [hunk 1]
```diff
@@ -2,13 +2,13 @@
 default:
 	$(MAKE) all
 test:
-	bash -c "./scripts/test.sh $(TEST)"
+	CGO_ENABLED=1 go test -v -race $(go list ./... | grep -v /vendor/)
```
Comment (woodsaj) [inline Makefile]: why are you enabling CGO, shouldn't it be disabled as is the case for building

Comment (Dieterbe) [inline Makefile]:
```

**Pull request merged**